components/raftstore/src/store/fsm/peer.rs

```diff
@@ -133,6 +133,9 @@ enum DelayReason {
 /// in most case.
 const MAX_REGIONS_IN_ERROR: usize = 10;
 const REGION_SPLIT_SKIP_MAX_COUNT: usize = 3;
+/// Limits the request size that can be batched in a single RaftCmdRequest.
+// todo: this fugure maybe changed to a more suitable value.
+const DEFAULT_MAX_BATCH_SIZE_LIMIT: u64 = 1 * 1024 * 1024;
```

Comment:
```suggestion
const MAX_BATCH_SIZE_LIMIT: u64 = 1 * 1024 * 1024;
```

Reply:
Seems your opinions are different. @LykxSassinator

Reply:
I will change back to MAX_BATCH_SIZE_LIMIT as it is not configurable so no "default" semantics.